[2025-04-28T10:35:20.341+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2025-04-08T00:00:00+00:00 [queued]>
[2025-04-28T10:35:20.479+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2025-04-08T00:00:00+00:00 [queued]>
[2025-04-28T10:35:20.485+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-04-28T10:35:20.486+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-04-28T10:35:20.488+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-04-28T10:35:20.730+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2025-04-08 00:00:00+00:00
[2025-04-28T10:35:20.922+0000] {standard_task_runner.py:55} INFO - Started process 111 to run task
[2025-04-28T10:35:20.995+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2025-04-08T00:00:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpudc8qdsr']
[2025-04-28T10:35:21.008+0000] {standard_task_runner.py:83} INFO - Job 229: Subtask create_table
[2025-04-28T10:35:24.342+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2025-04-08T00:00:00+00:00 [running]> on host fae2b1eff5d6
[2025-04-28T10:35:26.727+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2025-04-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-04-08T00:00:00+00:00
[2025-04-28T10:35:27.850+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2025-04-28T10:35:27.923+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstName TEXT NOT NULL,
                lastName TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2025-04-28T10:35:27.981+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2025-04-28T10:35:28.484+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20250408T000000, start_date=20250428T103520, end_date=20250428T103528
[2025-04-28T10:35:29.601+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-04-28T10:35:32.488+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
